src/components/forms/Range/Range.types.ts

```diff
@@ -0,0 +1,19 @@
+export interface RangeProps
```

Comment:
Could you please add short description of props like we have in https://github.com/securityscorecard/design-system/blob/alpha/src/components/Pill/Pill.types.ts#L45-L64
Storybook is able to pick it up and display in the props table